my-ft-digest-promo/promo-message.html

```diff
@@ -0,0 +1,38 @@
+<div class="n-myft-digest-promo">
+
+	<div class="o-grid-container n-myft-digest-promo-dismiss-btn__container">
+		<div class="o-grid-row">
+			<div data-o-grid-colspan="12" class="n-myft-digest-promo-dismiss-btn__content">
+				<button class="n-myft-digest-promo__dismiss-btn" data-trackable="digest-promo-dismiss"></button>
+			</div>
+		</div>
+	</div>
+
+		<div class="o-grid-container n-myft-digest-promo__container">
+			<div class="o-grid-row">
+				<div data-o-grid-colspan="12 M8 L10" class="n-myft-digest-promo__content">
+					<h2 class="n-myft-digest-promo__heading">
+					  Add to <nobr><span style="font-weight:normal">myFT Digest</span></nobr>
```

Comment:
Oh wow TIL what a `<nobr>` is. MDN suggests it's best to go for `white-space: nowrap` over this though: https://developer.mozilla.org/en/docs/Web/HTML/Element/nobr